package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@grrr/gulpfile",
-  "version": "9.0.1",
+  "version": "9.0.2",
```

Comment:
Did you publish this already with `yarn publish`? Otherwise you should not edit this version number by hand.
`yarn publish` will do it for you.

Reply:
No I edited the version number in the package. So will drop the last commit and do this with `yarn publish`